pkg/provider/provider.go

```diff
@@ -639,7 +640,8 @@ func (p *Provider) buildHTTPServiceFromTrafficTarget(t *topology.Topology, tt *t
 	for i, podKey := range tt.Destination.Pods {
 		pod, ok := t.Pods[podKey]
 		if !ok {
-			p.logger.Errorf("Unable to find Pod %q", podKey)
+			p.logger.Errorf("Unable to find Pod %q for HTTP service from Service Traffic Target %s@%s", podKey, topology.Key{Name: tt.Name, Namespace: tt.Namespace})
```

Comment:
```suggestion
			p.logger.Errorf("Unable to find Pod %q for HTTP service from Traffic Target %s@%s", podKey, topology.Key{Name: tt.Name, Namespace: tt.Namespace})
```

Reply:
would `from target:` be unambiguous enough for the purpose of keeping the log volume to a minimum yet provide sufficient context?